### ISSUE TITLE
Release 4.2 - Import

### ISSUE THREAD

**Issue by RemcoSimonides** [blockframes/blockframes#8307]
[Import test scenarios](https://www.notion.so/cascade8/Import-78250f00339a4aa9a2983c03d85a0b6f)

Issues resolved for this release:
- [x] https://github.com/blockframes/blockframes/issues/8075
- [x] https://github.com/blockframes/blockframes/issues/8270
- [x] https://github.com/blockframes/blockframes/issues/8239

**Comment by RemcoSimonides** [comment]
- [ ] In the CRM, page /crm/import/contracts when we import a document with errors, the modal doesn't have the cross to close it. And if you click outside the modale it doesn't remove it.

![image](https://user-images.githubusercontent.com/93206363/166223176-22910f43-4289-4635-a68f-9bf19f9d7e2f.png)

**Comment by phpgeekfr** [comment]
> In the CRM, page /crm/import/contracts when we import a document with errors, the modal doesn't have the cross to close it. And if you click outside the modale it doesn't remove it.


@KevinDurand974

**Comment by phpgeekfr** [comment]
**Organization import:**
- [ ] when importing single or multiple orgs, we don't know if process is running org not until the snackbar at the end. Button should be disabled with a spinner
- [ ] After submitting, single create button is not always disabled. If I try to create again, since email for org admin already exists I got an error and a better snackbar should be displayed
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/39384be8-0a98-4e62-98d8-820254f60f18)

-  [ ] even if orgs are imported in `accepted` status, we sill have the mail `Archipel Market - Bruce A2 was created and needs a review`

**Comment by phpgeekfr** [comment]
**Title import**

- [ ] after importing 2 movies, the snackbar said that 2/2 where imported but the button is still in this state :
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/48ae8c96-e05b-49db-8267-f16bc4af4646)

- [ ] paginator not working
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/122da596-8090-4504-9df6-7c6587a655e5)

- [ ] a title imported with "content type" movie is showing warnings about "season number"
 ![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/33e2b8f4-3bed-48ed-8d8e-23d50e2f6f22)

- [ ] errors on available versions even if I use data provided by excel  
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/a052a5d1-ade8-4227-bd37-dbf40d0eaa39)

- [ ] it should be "an admin template"
 ![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/cf56fa24-a78c-4623-bb51-af6ad4324f9b)

- [ ] with not blockframes admin user, imported titles from `/c/o/dashboard/import` are not displayed as draft in movie list. Access is set to false :
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/b36dfec8-41ac-4699-b807-153024775712)

- [ ] according to  https://www.notion.so/cascade8/Titles-17348f84c55740e1818332af889389f0 : `Try importing the excel template on the app with admin columns filled (warning should show about admin columns)` but instead of a warning, an error is displayed

**Comment by phpgeekfr** [comment]
**Contract import**

- [ ] paginator is not working 
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/4ae7a513-c6c7-41e5-a98f-f490724d2a15)

- [ ] Template is `xlsm` and we have a sentence : `this template contains an Excel Macro. Please enable it before using the document.` but accepted formats are still :
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/382e7a30-19b2-4442-bb0f-53d2e424b562)

- [ ] Template says `Purple cells are for admins only.` but there is no purple cells.

**Comment by n-germain-work** [comment]
# Movie import

- [ ] `Original title` is treated as mandatory, whereas it is not in the movie tunnel
![image](https://user-images.githubusercontent.com/78768220/166839857-502c9120-1720-462a-ba41-7c925a19979e.png)
vs
![image](https://user-images.githubusercontent.com/78768220/166839879-9b917d6d-8d9c-43a5-8fac-72e3c69cf6f8.png)

- [ ] it would feel nice to have an auto-scroll in the modal. Right now we have to do it manually if the next or previous item is out of the window
![Modal - auto-scroll](https://user-images.githubusercontent.com/78768220/166840186-6e7a0ab3-3f00-485e-a978-306942642ea0.gif)

# Contracts

- [ ] `Licensee` field is treated as required, but it is `complementary information` in the template

**Comment by RemcoSimonides** [comment]
Found by @Poppe-A; could you provide the file you used for import too, Antoine?

- [ ] (AAAAA) Import is failing with some field :
- Keyword & target audience 
- Social responsibility (fields with list of choices in template fields)
![import](https://user-images.githubusercontent.com/35397614/166455846-aef9aabc-f4ca-41b8-87e4-9dd3a1c8d76c.png)

**Comment by phpgeekfr** [comment]
- [ ] when uploading movie, excel template does not have director "filmography" . If we go to marketplace preview, there is a snackbar error saying: 
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/09ef6efa-0b79-4e76-8596-978b9efab5b9)

This can be tested with `bruce movie` from [https://app.zenhub.com/files/165818079/32afe9d4-e22c-4b8f-b3d5-f3224237ec03/download](https://app.zenhub.com/files/165818079/32afe9d4-e22c-4b8f-b3d5-f3224237ec03/download)

**Comment by cams-goussale** [comment]
@yspringfield 

- [ ] Could we upload new templates of the Movie import? I've updated the 2 templates [here](https://drive.google.com/drive/folders/1hmsmslJxLxrPpA5Cy5K814Rgd_bHYaQb) for Admin and for User. Here are the changes I made:

- The media parameters were wrong and different from the movie tunnel, so I've updated them.
- The templates available to download on the app (Archipel app and CRM) were not the latest versions that we discussed in this issue #8270


Also please note that some of the reviews can be disregarded because they are a consequence of me not updating the documentation. The notion documentation is not 100% up to date with the new templates formats.

> (C, this is camille's mistake) according to https://www.notion.so/cascade8/Titles-17348f84c55740e1818332af889389f0 : Try importing the excel template on the app with admin columns filled (warning should show about admin columns) but instead of a warning, an error is displayed

> (C, also camille's mistake) it should be "an admin template"

> (C) Template says Purple cells are for admins only. but there is no purple cells.
Yes there is, column C

**Comment by cams-goussale** [comment]
- [ ] @yspringfield could we upload new templates for the Contract import? I've updated the 2 templates [here](https://drive.google.com/drive/folders/1S_fi8-jmVcRSGAurrI4zCg49KCsnTnDo) for Admin and for User. Here are the changes I made:

- I've made the column Licensee name yellow because it is treated as mandatory
- I've removed the sentence about purple cells.

> C) Licensee field is treated as required, but it is complementary information in the template

> (C) Template says Purple cells are for admins only. but there is no purple cells.
